### PR TITLE
Inline placeholder

### PR DESCRIPTION
**Pull request by caillou**
This closes WCMS-910.

### CODE REVIEW

**Comment by caillou** [comment]
@peyerluk ready for review

File: src/rendering/component_view.coffee [hunk 1]
```diff
@@ -213,20 +213,12 @@ module.exports = class ComponentView
 
       $elem.removeClass(config.css.emptyImage)
     else
-      setPlaceholder = $.proxy(@setPlaceholderImage, this, $elem, name)
+      setPlaceholder = $.proxy(@setPlaceholderImage, this, name)
       @delayUntilAttached(name, setPlaceholder) # todo: replace with @afterInserted -> ... (something like $.Callbacks('once remember'))
 
 
-  setPlaceholderImage: ($elem, name) ->
-    $elem.addClass(config.css.emptyImage)
```
Comment (marcbachmann) [inline src/rendering/component_view.coffee]: We don't set a doc-image-empty class anymore?

Comment (caillou) [inline src/rendering/component_view.coffee]: Good catch! I accidentally deleted that line.

**Comment by caillou** [comment]
@marcbachmann re-added the missing CSS class. Ready for review.

**Comment by marcbachmann** [comment]
Looks good for now.
Is it possible to move the base64 string into a configuration variable? Or do you think we're too early in development to do this?

**Comment by caillou** [comment]
@marcbachmann moving it into a config file could make sense. I guess the best would be to have an SVG that is transformed to base64 using grunt. WDYT?

**Comment by marcbachmann** [comment]
For now we don't have to convert it automatically. We can do that manually and write a one-liner in the config.

**Comment by marcbachmann** [comment]
@rawdiggie I wanted to overwrite the `background-size` of the empty image placeholder. But sadly it get's overwritten by some design styles. How do we handle such styles? !important?
screen shot 2015-01-16 at 22 39 20

**Comment by marcbachmann** [comment]
@peyerluk The placeholders are working and this is ok to merge.
There's only the small problem with the css.
placeholders

**Pull request merged**